exopy_hqc_legacy/instruments/drivers/dll/sp_adq14.py

```diff
@@ -216,6 +222,8 @@ def get_traces(self, channels, duration, delay, records_per_capture,
         get_data = self._dll.GetData.func
         retrieved_records = 0
         while retrieved_records < records_per_capture:
+            # WHY ???
```

Comment:
I would really like to know why this is here. At least we should revert this before merging.

Reply:
moved to the other pull request with random stuff